tip-wlan/charts/wlan-ssc-service/templates/deployment.yaml

```diff
@@ -45,7 +45,7 @@ spec:
                 name: {{ include "common.fullname" . }}-creds
                 key: schema-repo-password
           image: cassandra:3.11.6
```

Comment:
While we are at it, it may be worthwhile to remove the hardcoded image and version and define these in values.yaml.